package.json

```diff
@@ -10,24 +10,26 @@
   },
   "private": true,
   "dependencies": {
-    "react": "^16.8.6"
+    "react": "^16.8.6",
+    "vega-embed": "^4.0.0"
```

Comment:
This probably should go into the package specific `package.json` as `peerDependencies` and `devDependencies` in the root `package.json`.